app/components/hero-component/index.jsx

```diff
@@ -21,8 +21,8 @@ function HeroComponent() {
         <div className="hero-overlay bg-opacity-60"></div>
 
         <div className="hero-content text-neutral-content">
-          <div className=" w-9/12">
-            <h1 className=" box-content mb-5 text-center lg:text-left text-7xl font-bold">
+          <div className="w-9/12">
+            <h1 className="box-content mb-5 text-center lg:text-left sm:text-3xl md:text-5xl lg:text-7xl font-bold">
```

Comment:
keep in mind, tailwind is mobile first sm is use for devices with min  with 640px

Breakpoint prefix | Minimum width | CSS
-- | -- | --
sm | 640px | @media (min-width: 640px) { ... }
md | 768px | @media (min-width: 768px) { ... }
lg | 1024px | @media (min-width: 1024px) { ... }
xl | 1280px | @media (min-width: 1280px) { ... }
2xl | 1536px | @media (min-width: 1536px) { ... }

<br class="Apple-interchange-newline">

https://tailwindcss.com/docs/responsive-design#targeting-mobile-screens 

Don’t use sm: to target mobile devices

Reply:
okey